gtk2_ardour/wscript

```diff
@@ -509,7 +509,6 @@ def build(bld):
         obj.source    = list(gtk2_ardour_sources)
         obj.target    = 'luadoc'
         obj.includes  = ['.', '../libs']
-        obj.ldflags   = ['-no-undefined']
```

Comment:
What is the motivation to remove `-no-undefined` LDFLAG?

Did this sneak in by accident, or is there some relation to python3 compat?

Reply:
I was unable to compile with this added. I'm not entirely sure why that is. Potentially there was a concatenation problem.

Reply:
As context, this is what happens, when the those flags are still there:

```
[ 685/1125] Linking build/tools/bb/bb
g++: error: unrecognized command line option ‘-no-undefined’
```
The verbose output is:

```
Build failed
 -> task in 'bb' failed with exit status 1: 
        {task 107607835508512: cxxprogram bb.cc.1.o,gui.cc.1.o,misc.cc.1.o -> bb}
['/usr/bin/g++', '-pthread', 'tools/bb/bb.cc.1.o', 'tools/bb/gui.cc.1.o', 'tools/bb/misc.cc.1.o', '-o/home/dave/Downloads/ardour/build/tools/bb/bb', '-Wl,-Bstatic', '-Wl,-Bdynamic', '-Llibs/ardour', '-Llibs/midi++2', '-Llibs/evoral', '-Llibs/audiographer', '-Llibs/ptformat', '-Llibs/pbd', '-Llibs/temporal', '-lardour', '-lmidipp', '-levoral', '-laudiographer', '-lptformat', '-lpbd', '-ltemporal', '-ljack', '-lgtkmm-2.4', '-latkmm-1.6', '-lgdkmm-2.4', '-lgiomm-2.4', '-lpangomm-1.4', '-lglibmm-2.4', '-lcairomm-1.0', '-lsigc-2.0', '-lgtk-x11-2.0', '-lgdk-x11-2.0', '-lpangocairo-1.0', '-latk-1.0', '-lcairo', '-lgdk_pixbuf-2.0', '-lgio-2.0', '-lpangoft2-1.0', '-lpango-1.0', '-lgobject-2.0', '-lglib-2.0', '-lharfbuzz', '-lfontconfig', '-lfreetype', '-no-undefined']
```